tests/std/test.lst

```diff
@@ -235,6 +235,7 @@ tests\P0769R2_shift_left_shift_right
 tests\P0784R7_library_support_for_more_constexpr_containers
 tests\P0811R3_midpoint_lerp
 tests\P0896R4_P1614R2_comparisons
+tests\P0896R4_P1614R2_spaceship

```

Comment:
No idea why I didn't comment on this before, but it seems odd that this test has a `P0896R4_` prefix when it doesn't seem to be testing anything from WG21-P0896. Should this be named just `tests\P1614R2_spaceship`?

Reply:
Good catch.